.pre-commit-config.yaml

```diff
@@ -42,9 +42,16 @@ repos:
       - id: mypy
         files: jupyter_client
         stages: [manual]
-        args: ["--install-types", "--non-interactive"]
+        args: []
         additional_dependencies:
-          ["traitlets>=5.13", "ipykernel>=6.26", "jupyter_core>=5.3.2"]
+          - traitlets>=5.13
+          - ipykernel>=6.26
+          - jupyter_core>=5.3.2
+          - types-pexpect
+          - types-paramiko
+          - types-netifaces
+          - types-psutil
+          - types-python-dateutil
```

Comment:
This enables `prek` to work, since it doesn't have pip installed, and `mypy` assumes `pip` is available to install things. And it's faster.